package.json

```diff
@@ -47,6 +47,7 @@
 		"lodash": "^4.17.20",
 		"markdown-it": "^12.0.4",
 		"markdown-it-task-lists": "^2.1.1",
+		"markdown-it-for-inline": "^0.1.1",
```

Comment:
From this plugins readme:

> This plugin allows to apply function to certain types of inline tokens. Speed will be not fastest of possible, but you can do quick prototyping of certain rule types.

I'd rather pick https://www.npmjs.com/package/markdown-it-link-attributes for this

Reply:
Not an expert at all, just looked into it because I would also like to have that feature. Would
https://github.com/markdown-it/markdown-it/blob/master/docs/architecture.md#renderer
also be a possibility without importing another library?